cmd/spx/template/game.js

```diff
@@ -70,93 +135,24 @@ class GameApp {
         const evt = new CustomEvent('add_files', {
             detail: datas
         });
-        this.gameCanvas.dispatchEvent(evt);
+        this.editorCanvas.dispatchEvent(evt);
```

Comment:
Should we wait for editorCanvas to finish processing event `add_files`?

Reply:
Yes, this is a TODO. We should wait for `editorCanvas `to finish processing the event after calling `this.deleteFiles`. It's a bit complex and involves significant changes, so I hope to implement it in the next few commits.

Reply:
this has been resove by https://github.com/goplus/spx/pull/378